packages/manager/apps/web/client/app/domain/domain.bundle.js

```diff
@@ -95,3 +91,10 @@ import './zone/record/delete/domain-zone-record-delete.controller';
 import './zone/record/domain-zone-record.controller';
 import './zone/reset/domain-zone-reset.controller';
 import './zone/zone.routing';
+
+import './zone/activate/activate.module';
```

Comment:
Ok to remove but not to add import from `*.bundle.js`
Maybe you could import `zone/activate/index` in `zone/activate` module ?